lib/modules/onc_program/test/onc_ehr_launch_sequence_test.rb

```diff
@@ -80,21 +80,29 @@
     end
 
     it 'fails when a badly formatted scope was received' do
-      bad_scopes = ['user/*/*', 'patient/*.read', 'user/*.*.*', 'user/*.write']
+      scopes = ['abc', 'patient/*/*', 'patient/.', 'patient/*.', 'patient/*.*.*', 'patient/*.write']
```

Comment:
shouldn't we allow any extra scope to be granted now (e.g. `abc`)?  Or is this not failing on the 'abc', but rather on one of those strangely formatted other scopes?